Source/JavaScriptCore/runtime/IntlDateTimeFormat.cpp

```diff
@@ -888,7 +888,7 @@ void IntlDateTimeFormat::initializeDateTimeFormat(JSGlobalObject* globalObject,
                     return;
                 }
                 replaceHourCycleInSkeleton(skeleton, specifiedHour12);
-                dataLogLnIf(IntlDateTimeFormatInternal::verbose, "replaced:(", StringView(skeleton.data(), skeleton.size()), ")");
+                dataLogLnIf(IntlDateTimeFormatInternal::verbose, "replaced:(", StringView(skeleton.span()), ")");
```

Comment:
Can just pass a `span` directly to `makeString`, no need to use a `StringView`.

Reply:
It didn't work.

Reply:
OK, I guess `dataLog` doesn’t use `StringConcatenate.h` maybe?